app/payment-request/get-payment-request.js

```diff
@@ -32,6 +32,12 @@ const getPaymentRequest = async (categoryId = [ENRICHMENT, LEDGER_ENRICHMENT]) =
     ],
     order: [['received']]
   })
+  for (let i = 0; i < paymentRequest.length; i++) {
+    if (paymentRequest[i].schemes?.name === 'SFI') {
+      paymentRequest[i].schemes.name = 'SFI22'
+    }
+  }
```

Comment:
You're doing this a few times, maybe best to write a function to do it?

Alternatively you could probably use `sequelize.fn` to use `replace` on the value in the query itself.

https://sequelize.org/api/v7/variables/_sequelize_core.index.sql.fn.html

Not 100% sure of the syntax though.